osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -35,11 +52,80 @@ private void load(OsuConfigManager osuConfig, FrameworkConfigManager config)
                     Bindable = osuConfig.GetBindable<bool>(OsuSetting.MouseDisableButtons)
                 },
             };
+
+            rawInputToggle.ValueChanged += enabled =>
+            {
+                // this is temporary until we support per-handler settings.
+                const string raw_mouse_handler = @"OpenTKRawMouseHandler";
+                const string standard_mouse_handler = @"OpenTKMouseHandler";
+
+                activeInputHandlers.Value = enabled ?
+                    activeInputHandlers.Value.Replace(standard_mouse_handler, raw_mouse_handler) :
+                    activeInputHandlers.Value.Replace(raw_mouse_handler, standard_mouse_handler);
+
+                sensitivity.Bindable.Disabled = !enabled;
+            };
+
+            rawInputToggle.TriggerChange();
+        }
+
+        private class SensitivitySetting : SettingsSlider<double, SensitivitySlider>

```

Comment:
we can move it if it comes up again. i couldn't think of another use case and didn't want to pollute `SliderBar`